doc/config_options.txt

```diff
@@ -40,6 +40,12 @@ The number of seconds to wait after the instance started before starting the nex
 The instance with the highest value is started first.
 ```
 
+```{config:option} boot.debug_edk2 instance-boot
+:shortdesc: "Enable debug version of the `edk2`"
+:type: "string"
```

Comment:
Is it a string or a bool?

Reply:
> Is it a string or a bool?

It's string. I was confused, because `volatile.apply_nvram` for some reason has type `string` while the value in it is always `bool`.

Reply:
volatile.apply_nvram is a bool from the perspective of the configuration options, because it can only be "true" or "false". 
The actual Go data type is a string, but thats because all LXD config options are encoded as map[string]string.

So `boot.debug_edk2` should be documented as being a `bool` type from the perspective of the valid values for the configuration option.

Reply:
yep, that's what confuses me. Should we make `volatile.apply_nvram` to be a `bool` too then?

Reply:
If the only allowed values are true/false (which from what I can see is the case) then yes.

Reply:
Have fixed. And also changed `volatile.apply_nvram` type to `bool` in a separate commit.